admin-api-server/src/analytics/service/analytics.service.ts

```diff
@@ -209,22 +220,33 @@ LIMIT ${params.pageSize}
     }
 
     return {
-      data: qryRes.rows.map(
-        (row: any) =>
-          <Activity>{
-            id: Number(row['id']),
-            timestamp: Math.floor(row['timestamp'].getTime() / 1000),
-            kind: row['kind'],
-            from: row['from'],
-            to: row['to'],
-            tokenId: Number(row['tokenId']),
-            price: this.currencyService.convertToCurrency(
-              Number(row['price']),
-              BASE_CURRENCY,
-            ),
-            amount: Number(row['amount']),
-          },
-      ),
+      data: qryRes.rows.map((row: any) => {
+        const price = this.currencyService.convertToCurrency(
+          Number(row['price']),
+          BASE_CURRENCY,
+        );
+        return <Activity>{
+          id: Number(row['id']),
+          timestamp: Math.floor(row['timestamp'].getTime() / 1000),
+          kind: row['kind'],
+          from: row['from'],
+          to: row['to'],
+          tokenId: Number(row['tokenId']),
+          price,
+          edition_size: Number(row['edition_size']),
+          currency: row['currency'],
+          transaction_value:
+            price &&
+            row['conversion_rate'] &&
+            (
+              Math.round(Number(price) * Number(row['conversion_rate']) * 100) /
+              100
+            ).toFixed(2),
```

Comment:
Why the `Round(... *100) / 100` as well as the `.toFixed(2)`? I think just the `.toFixed(2)` suffices